src/main/java/org/generationcp/middleware/service/DataImportServiceImpl.java

```diff
@@ -1099,6 +1102,11 @@ public int saveProjectData(final Workbook workbook, final String programUUID, fi
 
 		this.checkForExistingTrialInstance(workbook, errors, programUUID);
 
+		// Validate Trial Instance Value
```

Comment:
As discussed, please check if you can put the numeric validation inside `checkForExistingTrialInstance` method (will have to rename to `validateTrialInstanceValues`)